### DISCUSSION TITLE
Delete observations ?

### DISCUSSION THREAD

**Discussion by alexbelgium** [tphakala/birdnet-go#129] [General]
Hi, I have birds wrongly identified. Is there an easy way to remove erroneous observations? Thanks!

**Comment by tphakala** [comment]
Unfortunately this is currently missing from web UI, I shortly had it in but didn't like UI layout so I removed UI elements until I figure out a cleaner way to present it. Backend code is there.

I also have a plan to allow reclassification for detections, so if you spot a detection which you know for sure to be specific species you could switch that from UI.

**Reply by alexbelgium** [reply]
Thanks! I do understand that it's a work in progress and thank you for all your work - I've been running it now for 2 days and it works perfectly well!
